From 9e9e2ccfb7542087e5bde827e5bad9f2553ad66a Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:26:35 +0200
Subject: [PATCH 2/2] formatting: use tabs instead of spaces

Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
---
 calculator.cpp | 3 +++
 calculator.h   | 6 +++---
 2 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/calculator.cpp b/calculator.cpp
index 30ce337..3203227 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,8 +2,11 @@
 
 int Calculator::Add (double a, double b)
 {
+<<<<<<< HEAD
     return a + b;
 	return a + b + 0.5;
+=======
+>>>>>>> cbcb06d... formatting: use tabs instead of spaces
 	return a + b;
 }
 
diff --git a/calculator.h b/calculator.h
index 3740907..d59d596 100644
--- a/calculator.h
+++ b/calculator.h
@@ -3,9 +3,9 @@
 
 class Calculator
 {
-    public:
-        int Add (double, double);
-        int Sub (double, double);
+	public:
+		int Add (double, double);
+		int Sub (double, double);
 };
 
 #endif//CALCULATOR_H
-- 
2.23.0.windows.1

